### PR TITLE
set focus to challenge when password already filled

### PR DESCRIPTION
**Pull request by e2e8**
- set focus to auth challenge field if it is present and the password has already been filled
- remove unnessary cygwin package mingw64-x86_64-gcc-g++

### CODE REVIEW

File: BUILD.rst [hunk 1]
```diff
@@ -22,7 +22,7 @@ their dependencies. You can install these packages using the standard
 - pkg-config
 - make
 - mingw64-x86_64-gcc-core
-- mingw64-x86_64-g++
+- mingw64-x86_64-gcc-g++
```
Comment (selvanair) [inline BUILD.rst]: As per cygwin package list ( https://cygwin.com/packages/package_list.html#m) this change is correct but why are we requiring c++/g++?  Could anyone using cygwin confirm we need this, and `mingw64-x86_64-gcc-core` is not enough?

Anyway, this chunk is unrelated and should be a separate commit or PR.

File: openvpn.c [hunk 2]
```diff
@@ -470,6 +470,10 @@ UserAuthDialogFunc(HWND hwndDlg, UINT msg, WPARAM wParam, LPARAM lParam)
             {
                 SendMessage(GetDlgItem(hwndDlg, ID_EDT_AUTH_PASS), EM_SETSEL, 0, MAKELONG(0,-1));
             }
+            else if (param->flags & FLAG_CR_TYPE_SCRV1)
+            {
+                SetFocus(GetDlgItem(hwndDlg, ID_EDT_AUTH_CHALLENGE));
+            }
```
Comment (selvanair) [inline openvpn.c]: Looks good.

File: BUILD.rst (hunk 1 above)
Comment (chipitsine) [reply BUILD.rst]: I can check it tomorrow

Comment (e2e8) [reply BUILD.rst]: I removed the g++ package and the project rebuilt successfully.

Comment (selvanair) [reply BUILD.rst]: Thanks for the follow up.
Yes, please cleanup the first commit (i.e., remove the editing of Build.rst from it) and force push. Then we can merge this.

Comment (chipitsine) [reply BUILD.rst]: I confirm, build is ok without g++ installed
